util/envutil/is_env_true.go

```diff
@@ -0,0 +1,18 @@
+package envutil
+
+import (
+	"os"
+)
+
+// IsEnvTrue returns true if the env variable `k` is set
+// to something bash would interpret as true.
+func IsEnvTrue(k string) bool {
+	v, found := os.LookupEnv(k)
+	if !found {
+		return false
+	}
+	if v == "" || v == "0" || v == "false" || v == "FALSE" {
+		return false
+	}
+	return true
```

Comment:
`!found` and `""` are being treated the same here, so we can just use `Getenv`.

Also a switch allows us to avoid repeating `v ==` for each case.

```suggestion
	switch os.Getenv(k) {
	case "", "0", "false", "FALSE":
		return false
	default:
		return true
	}
```